terraform/main.tf

```diff
@@ -349,6 +368,30 @@ module "github_secrets" {
       name  = "TF_AZURE_IDENTITY_2_APP_ID"
       value = module.azuread_applications.identity_2.client_id
     },
+    {
+      name  = "TF_AZURE_POSTGRES_IDENTITY_APP_ID"
+      value = module.azuread_applications.postgres_identity.client_id
+    },
+    {
+      name  = "TF_AZURE_POSTGRES_IDENTITY_NAME"
+      value = module.azuread_applications.postgres_identity.name
+    },
```

Comment:
After removing the postrgresql identity, we can jsut use TF_AZURE_IDENTITY_1_APP_ID